contracts/ExitHandler.sol

```diff
@@ -136,6 +136,42 @@ contract ExitHandler is DepositHandler {
     );
   }
 
+  function startDepositExit(uint256 _depositId) public payable {
+    require(msg.value >= exitStake, "Not enough ether sent to pay for exit stake");
+    // check that deposit exits
+    Deposit deposit = deposits[uint32(_depositId)];
+    require(deposit.owner == msg.sender, "Only deposit owner can start exit");
+    require(deposit.amount > 0, "deposit has no value");
+    require(exits[bytes32(_depositId)].amount == 0, "The exit of deposit has already been started");
+    require(!exits[bytes32(_depositId)].finalized, "The exit for deposit has already been finalized");
+    
+    uint256 priority;
+    if (isNft(deposit.color)) {
+      priority = (nftExitCounter << 128) | uint128(_depositId);
+      nftExitCounter++;
+    } else {      
+      priority = getERC20ExitPriority(uint32(deposit.time), bytes32(_depositId), 0);
+    }
+
+    tokens[deposit.color].insert(priority);
+
+    exits[bytes32(_depositId)] = Exit({
```

Comment:
How likely is it that we get a clash with traditional exits here? How are our utxoids assigned again?